### ISSUE TITLE
The master branch does not compile on Qt 5.6...

### ISSUE THREAD

**Issue by hazrpg** [team-phoenix/Phoenix#270]
There were several build errors when trying to compile using Ubuntu 16.04 and Qt 5.6.

I have created a patch file to show what I had to change to get it to compile - obviously my changes aren't recommended because I have simply changed info to debug (because I have no knowledge as to how to get info to properly work). However it seems in some places Q_ENUM was used incorrectly (it should have been Q_ENUMS).

Anyway here is the diff patch file:
[patch.txt](https://github.com/team-phoenix/Phoenix/files/299351/patch.txt)

If there's anything I can do to help, please let me know.

**Comment by athairus** [comment]
Hi, could you please post the full build log? I've built this myself on Ubuntu 16.04 with no issues only a week or two ago.

I'll build again to confirm.

**Comment by athairus** [comment]
Just as a hunch, do you have Qt 4 packages installed? They're known to interfere with Qt 5, at least with our project.

**Comment by athairus** [comment]
I've just rebuilt the project on 16.04 with no problem.

**Comment by hazrpg** [comment]
Hi @athairus, thank you very much for getting in touch so quickly. You are indeed correct, the conflict was due to Qt 4 packages that were installed.

Another thing I should probably mention too is that I've had QtCreator installed for quite some time (and I keep upgrade it as and when needed) - I had to scrap all my config files completely and uninstall everything and start again. Together, this seemed to resolve the issue.
